scoresheets/Restaurant.tex

```diff
@@ -3,19 +3,21 @@
 	\scoreitem[2]{100}{Detect calling or waving customer}
 	\scoreitem[2]{100}{Reach a customer's table without prior guidance/training}
 
-	\scoreitem[2]{300}{Take an order.}
-	\scoreitem[2]{300}{Serve an order.}
+	\scoreitem[2]{200}{Understand and confirm the order received to the customer}
+	\scoreitem[2]{100}{Communicate the order to the barman}
+
+	\scoreitem[2]{100}{Return to the customer table with the order}
+	\scoreitem[2]{200}{Serve the order byplacing the items on the table}
 
 	\scoreheading{Bonus Rewards}
 	\scoreitem[2]{200}{Use an unattached tray to transport}
 	
 	\scoreheading{Regular Penalties}
+	\penaltyitem[2]{200}{Being guided to a table}
 	\penaltyitem[2]{80}{Not making eye-contact when taking an order}
 	\penaltyitem[2]{80}{Not reaching the bar (barman has to move from behind the bar to interact with the robot)}
 
-
 	\scoreheading{Deus ex Machina Penalties}
-	\penaltyitem[2]{200}{Being guided to a table}
 	\penaltyitem[4]{50}{Asking the Barman to handover object to the robot}
 	\penaltyitem[4]{50}{Guest needing to take the object from a tray or the robot's hand}
```

Comment:
Can we somehow merge this? 

The Barman handover also includes the barman putting it onto a tray or into a bag

We could have -25 handover (to/from gripper) and -50 for placing/getting from bag or tray but:
- i dont think anyone using handover instead of letting the people just grab it for 25 points
- The robot pulling the item from an attached tray/bag is, for me, more impressive than e.g. pick/place with a dual manipulator robot.

.. Just keep it as is :sweat_smile: